rllib/algorithms/tests/test_algorithm_config.py

```diff
@@ -192,6 +194,59 @@ class A:
         config.validate()
         self.assertEqual(config.rl_module_spec.module_class, A)
 
+    def test_learner_hyperparameters_per_module(self):
```

Comment:
New test case, checking the correct `algorithm_config_override` propagation into the LearnerHPs.